packages/linea-ens-resolver/contracts/L1Resolver.sol

```diff
@@ -237,14 +260,19 @@ contract L1Resolver is
             .element(coinType)
             .fetch(
                 this.addrCoinTypeCallback.selector,
-                abi.encode(L2_BLOCK_RANGE_ACCEPTED)
+                abi.encode(ACCEPTED_L2_BLOCK_RANGE_LENGTH)
```

Comment:
given this is never actually read from the callbackData, do we even need it in the callbackData?

Reply:
I think it is good to keep it if some users wants to create a new contract that use it. WDYT ?

Reply:
Another completely different approach I'm thinking would be to add a constructor in `EVMFetchTarget`(Which is inherited by `L1Resolver`) with the l2BlockRangeAcceptedLength as argument and set it there when deploying `L1Resolver`. 
That way it stays generic for other contracts that inherits it to set whatever range they are confortable with.

What do you think ?

Reply:
> I think it is good to keep it if some users wants to create a new contract that use it. WDYT ?

Yes that's why we need to keep it

Reply:
if they were deploying their own modified version, they can just change the constant - don't see the need to add constructor complexity

Reply:
evidently it was me misreading the unexpanded view of the function - it is actually 2 functions..

Reply:
So what do you think about this approach ?

Reply:
Instead of the constructor I could add a virtual function in `EVMFetcher` to be overridden in the child contract (here `L1Resolver`)